ansible/roles/pmm2-images/tasks/main.yml

```diff
@@ -28,11 +28,10 @@
     - name: Packages                   | Install OS tools
       yum:
         name:
-          - screen
+          - lsof
```

Comment:
screen is almost useless inside the container; lsof is often needed for debugging open ports

Reply:
well, arguably not useless as it saves you connecting multiple times